deploy/base/deployment.yaml

```diff
@@ -66,13 +66,15 @@ spec:
               path: /healthz
               port: 8080
             failureThreshold: 30
-            periodSeconds: 10
```

Comment:
Is this deleted because the default is 10?

Reply:
Yes, default is 10